charts/templates/configmap-script.yaml

```diff
@@ -0,0 +1,663 @@
+{{- $replicas := .Values.replicaCount }}
+{{- $releaseName := include "marklogic.fullname" . }}
+{{- $namespace := .Release.Namespace }}
+{{- $clusterDomain := .Values.clusterDomain }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "marklogic.fullname" . }}-scripts
+  namespace: {{ .Release.Namespace }}
+data:
+  poststart-hook.sh: |
+    #! /bin/bash
+    ###############################################################
+    #
+    #   Copyright 2023 MarkLogic Corporation.  All Rights Reserved.
+    #
+    ###############################################################
+    #   Group Configuration MarkLogic server
```

Comment:
I'm not sure that the name Group Configuration MarkLogic server is relevant here.
We should put Initialise and configure MarkLogic cluster.

Reply:
Removed these comments for now